bitnami/redis/templates/configmap-scripts.yaml

```diff
@@ -22,10 +24,35 @@ data:
             false
         fi
     }
+    retry_while() {
+        local cmd="${1:?cmd is missing}"
+        local retries="${2:-12}"
+        local sleep_time="${3:-5}"
+        local return_value=1
+        read -r -a command <<< "$cmd"
+        for ((i = 1 ; i <= retries ; i+=1 )); do
+            "${command[@]}" && return_value=0 && break
+            sleep "$sleep_time"
+        done
+        return $return_value
+    }
+    not_exists_dns_entry() {
+      myip=$(hostname -i)
+
+      if [[ -z "$(getent ahosts "$HEADLESS_SERVICE" | grep "^${myip}" )" ]]; then
+        echo "$HEADLESS_SERVICE is not resolving or not has my IP: ${myip}"
```

Comment:
$HEADLESS_SERVICE does not contain the IP of this pod: ${myip}

Reply:
Fixed